### PR TITLE
move event attributes to the registry

### PR DESCRIPTION
**Pull request by mlunadia**
This PR moves event attributes to the registry. It is part of https://github.com/open-telemetry/semantic-conventions/issues/407

### CODE REVIEW

File: docs/general/events.md [hunk 1]
```diff
@@ -43,18 +43,10 @@ that identify the class of Events but not the instance of the Event.
 <!-- semconv event -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| `event.domain` | string | The domain identifies the business context for the events. [1] | `browser` | Required |
-| `event.name` | string | The name identifies the event. | `click`; `exception` | Required |
+| [`event.domain`](../attributes-registry/events.md) | string | The domain identifies the business context for the events. [1] | `browser` | Required |
+| [`event.name`](../attributes-registry/events.md) | string | The name identifies the event. | `click`; `exception` | Required |
 
 **[1]:** Events across different domains may have same `event.name`, yet be unrelated events.
-
```
Comment (joaopgrassi) [inline docs/general/events.md:50]: Please render the table with `full` (check other examples) so we don't loose the enum table values

**Review by joaopgrassi — CHANGES_REQUESTED** [review]

File: docs/attributes-registry/events.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Events
```
Comment (trisch-me) [inline docs/attributes-registry/events.md:2]: Should it be `events` or `event`?

File: docs/general/events.md (hunk 1 above)
Comment (mlunadia) [reply docs/general/events.md]: as `event.domain` has been removed it looks like we don't need the `full` render anymore

**Review by ChrsMark — APPROVED** [review]
lgtm!

@joaopgrassi could you have another look?

**Pull request closed without merging**